open-machine-learning-jupyter-book/_toc.yml

```diff
@@ -83,23 +87,22 @@ parts:
     - file: ml-advanced/gradient-boosting/xgboost.ipynb
     - file: ml-advanced/gradient-boosting/xgboost-k-fold-cv-feature-importance.ipynb
   - file: ml-advanced/unsupervised-learning
-  - file: ml-advanced/kernel-method
   - file: ml-advanced/model-selection
```

Comment:
I think line 86 should be retained, and add a line of `- file: ml-advanced/unsupervised-learning-pca-and-clustering.ipynb`

Reply:
DONE